core/server/data/migrations/versions/3.25/02-add-amp-gtag-id-setting.js

```diff
@@ -0,0 +1,20 @@
+const logging = require('../../../../../shared/logging');
+
+module.exports = {
+    config: {
+        transaction: true
+    },
+
+    async up({transacting: knex}) {
+        logging.info('Updating amp_gtag_id setting to amp group');
+        await knex('settings')
+            .update({
+                group: 'amp'
```

Comment:
`flags` can be left as default `null`

Reply:
if the aim is to be completely detached from model layer and any other env, wouldn't it make sense to have ll the fields explicitly assigned tho?

Reply:
According to this comment, we should leave it as it is https://github.com/TryGhost/Ghost/pull/12039#discussion_r454563140